app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str | Exception:
+    masks_to_buy = 0
+    quantity_of_friends = 0
```

Comment:
There is no need for the `quantity_of_friends` variable and logic related to it. Remove it from your code.